packages/camera/camera_platform_interface/lib/src/types/video_capture_options.dart

```diff
@@ -43,6 +44,17 @@ class VideoCaptureOptions {
   /// Should only be set if a streamCallback is also present.
   final CameraImageStreamOptions? streamOptions;
 
+  /// **Android only.**
+  ///
+  /// When `true`, configures the recording to be a persistent recording.
+  ///
+  /// A persistent recording will only be stopped by explicitly calling [CameraController.stopVideoRecording] and will ignore events that would normally cause recording to stop, such as
+  /// * lifecycle events
+  /// * calling [CameraController.setDescription] while a recording is in progress
+  ///
+  /// Defaults to `false`.
+  final bool enableAndroidPersistentRecording;
```

Comment:
Platform interfaces should not be platform-specific. What if another platform needs this configuration in the future?

The name and documentation should be sufficiently generic that it could apply to any relevant platform, even if not all platforms require it.